vignettes/dose-prob_funs.Rmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "Example of usage of dose and prob functions"
```

Comment:
can you please make this more general? I would say e.g. "Migrating from crmPack version < ..." (input current CRAN version)
and then the dose/prob is a section in here. thereby we can add other topics later and don't get too many vignettes.